jerry-libc/include/time.h

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ * Copyright 2016 University of Szeged
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef JERRY_LIBC_TIME_H
+#define JERRY_LIBC_TIME_H
+
+#ifdef __cplusplus
+# define EXTERN_C "C"
```

Comment:
I agree with @zherczeg here. It would be better to use the more traditional
```c
#ifdef __cplusplus
extern "C" {
#endif
// header main content here
#ifdef __cplusplus
}
#endif
```
approach. The issue is that the `#define EXTERN_C "C"` way is used all over the project. I'd love to see that change.